zebra-chain/Cargo.toml

```diff
@@ -24,6 +24,11 @@ json-conversion = [
     "serde_json",
 ]
 
+# Async error handling convenience traits
+async-error = [
```

Comment:
Why the new feature?

Reply:
Because some developers might want to depend on a parsing crate that doesn't depend on tokio. (I think we've had this request or concern before.)